challenge.js

```diff
@@ -10,6 +10,8 @@
 
 // ANSWER
 
+You would run a JS function that calls the document.createElement() method.
+
```

Comment:
That's halfway there - it will create an element, but it won't add it to the page. You'll need to take that new element and add it to an existing element in the DOM.